tests/attr/test_saliency.py

```diff
@@ -76,27 +85,21 @@ def _saliency_base_assert(
                 stdevs=0.0000002,
                 additional_forward_args=additional_forward_args,
             )
-        if isinstance(attributions, tuple):
-            for input, attribution, expected_attr in zip(
-                inputs, attributions, expected
-            ):
-                if nt_type == "vanilla":
-                    self._assert_attribution(attribution, expected_attr)
-                self.assertEqual(input.shape, attribution.shape)
-        else:
+
+        for input, attribution, expected_attr in zip(inputs, attributions, expected):
```

Comment:
I found that the if/else check is redundant in this case -- no matter the `attributions` is a tuple or not, we can use one for loop to iterate over. In addition, this refactor makes mypy happy with types of attributions/inputs